code/modules/jobs/job_types/gimmick.dm

```diff
@@ -20,8 +20,12 @@
 	rpg_title = "Peasant"
 	allow_bureaucratic_error = FALSE
 	outfit = /datum/outfit/job/gimmick
+	species_outfits = list(
+		SPECIES_PLASMAMAN = /datum/outfit/plasmaman
+	)
```

Comment:
Moved `species_outfits` to the main type, and removed other instances that used the same outfit because there was no unique one available (Barber and Psychiatrist)